src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -97,8 +97,8 @@
  * </pre>
  * <p>
  * The general rule is that the argument of the {@code @SuppressWarnings} will be
- * matched against class name of the checker in lower case and without {@code Check}
- * suffix if present.
+ * matched against class name of the checker in any letter case. Adding {@code check}
```

Comment:
> checker 

I am assuming this is suppose to be check or module.

Reply:
That is correct.

Reply:
@stoyanK7 , please fix this typo "checker" -> "Check".

Reply:
Done!

Reply:
Checks are now passing too!

Reply:
@rnveach ping